src/NRedisStack/Json/JsonCommandsAsync.cs

```diff
@@ -82,8 +82,8 @@ public async Task<long> DelAsync(RedisKey key, string? path = null)
         var res = await _db.ExecuteAsync(JsonCommandBuilder.Get<T>(key, path));
         if (res.Type == ResultType.BulkString)
         {
-            var arr = JsonSerializer.Deserialize<JsonArray>(res.ToString()!);
-            if (arr?.Count > 0)
+            var arr = JsonSerializer.Deserialize<JsonElement[]>(res.ToString()!);
```

Comment:
It's because you are using an older version of System.Text.Json - I think you SHOULD be able to switch it to 7.0.2 (current version) without breaking compatibility. The best way to validate that would be to make sure that https://github.com/redis/NRedisStack/blob/df68194b262e75440fb87e78c0762f18458d96cf/tests/NRedisStack.Tests/NRedisStack.Tests.csproj#L4 points to some older versions of .NET Framework (e.g. 4.5.2)

Reply:
do you mean to change the Version in this line: `<PackageReference Include="System.Text.Json" Version="4.7.1" Condition="'$(TargetFramework)' == 'netstandard2.0'" />`
to 7.0.2?

Reply:
No - so in this case we want to also test against an additional framework (just to make sure that the library is going to be fully compatible with that framework) .NET 4.5.2 is the oldest supported framework in .NET Standard 2.0 - so you just add `net452` to the semi-colon delaminated list of target frameworks. You will need to enable this to run on the `windows-latest` runner in your [integration job](https://github.com/redis/NRedisStack/blob/master/.github/workflows/integration.yml#L21) since .NET Framework can't run on Ubuntu. And you'll need to adjust your `dotnet test` command to specify the frameworks with the `-f` option in each job. See docs [here](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-test)